helix-core/src/macros.rs

```diff
@@ -7,7 +7,7 @@ macro_rules! hashmap {
     ($($key:expr => $value:expr),*) => {
         {
             let _cap = hashmap!(@count $($key),*);
-            let mut _map = ::std::collections::HashMap::with_capacity(_cap);
+            let mut _map = std::collections::HashMap::with_capacity(_cap);
```

Comment:
This is deliberate for macro hygiene purposes.

Reply:
Mm sure, I can revert this. Even though I doubt there will be a namespace are namespace collisions with: `td::collections::HashMap::with_capacity`